### PR TITLE
Customer Portal Sessions 

### PR DESCRIPTION
**Pull request by danbillson**
# Customer portal sessions

Relevant docs: https://developer.paddle.com/api-reference/customer-portals/overview

Usage:
```
  const session = await paddle.customerPortalSessions.create(customerId, [
    subscriptionId,
  ]);
```

### CODE REVIEW

File: src/__tests__/mocks/resources/customer-portal-sessions.mock.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  meta: {
+    request_id: '',
+    pagination: {
+      estimated_total: 10,
+      has_more: true,
+      next: '/customers/ctm_123/portal-sessions?after=1',
+      per_page: 10,
+    },
+  },
+};
```
Comment (davidgrayston-paddle) [inline src/__tests__/mocks/resources/customer-portal-sessions.mock.ts:49]:
```suggestion
```

I don't think we will need this?

File: src/__tests__/resources/customer-portal-sessions.test.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    const customerPortalSessionsResource = new CustomerPortalSessionsResource(paddleInstance);
+    const createdCustomerPortalSession = await customerPortalSessionsResource.create(customerId, subscriptionIds);
+
+    expect(paddleInstance.post).toBeCalledWith(`/customers/${customerId}/portal-sessions`, {
+      subscriptionIds,
+    });
+    expect(createdCustomerPortalSession).toBeDefined();
+    expect(createdCustomerPortalSession.id).toBeDefined();
+    expect(createdCustomerPortalSession.customerId).toBe(customerId);
```
Comment (davidgrayston-paddle) [inline src/__tests__/resources/customer-portal-sessions.test.ts:29]: We could add some additional assertions for `urls` and `created_at` here?

**Review by davidgrayston-paddle — DISMISSED** [review]
Looks good 👍

File: src/__tests__/resources/customer-portal-sessions.test.ts (hunk 2 above)
Comment (danbillson) [reply src/__tests__/resources/customer-portal-sessions.test.ts:29]: Sounds good to me!

**Review by davidgrayston-paddle — APPROVED** [review]

**Pull request merged**